content/en/docs/contribute/style/style-guide.md

```diff
@@ -146,6 +146,33 @@ Set the value of the `replicas` field in the configuration file. | Set the value
 The value of the `exec` field is an ExecAction object. | The value of the "exec" field is an ExecAction object.
 {{< /table >}}
 
+### Use code style for Kubernetes command tool and component names
+
+{{< table caption = "Do and Don't - Use code style for Kubernetes command tool and component names" >}}
+Do | Don't
+:--| :-----
+The `kubelet` preserves node stability. | The kubelet preserves node stability.
+The `kube-proxy` maintains network rules on nodes. | The kube-proxy maintains network rules on nodes.
+Kubectl handles locating and authenticating to the API server. | `kubectl` handles locating and authenticating to the apiserver.
```

Comment:
This example falls into the "Don'ts" category below -- do not start a sentence with a tool name.

Reply:
@tengqm OK.
There are a number of places in the docs where a cmd/component name starts a sentence. I'll go through the examples on this page again. I am hoping to capture the cases where this may be a possible construction (start the sentence with a capitalized tool/exe name or use something like `The kubelet`). Do you think cmd tool and component _names_ should use `code` formatting within a sentence?

Reply:
I like the idea of using “The ” or similar text to avoid needing to write (eg) “Kube-proxy maintains rules”.